CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Change Log
 
+## [0.22.0](https://github.com/eclipse/lemminx/milestone/35?closed=1) (September 28, 2022)
+
+### Enhancements
+
+ * Suppress validation kinds based on file pattern. See [#1275](https://github.com/eclipse/lemminx/issues/1275).
+ * Fix CI badge, add some more badges. See [#1262](https://github.com/eclipse/lemminx/pull/1262).
+ * CodeAction for cvc-complex-type.2.4.b: "insert all expected" vs. "insert all possible". See [#1255](https://github.com/eclipse/lemminx/issues/1255).
+ * Use proxy config as HTTPS proxy as well. See [#1253](https://github.com/eclipse/lemminx/pull/1253).
+ * Support `xml.format.closingBracketNewLine` setting with experimental formatter. See [#1247](https://github.com/eclipse/lemminx/issues/1247).
+ * Support `xml.format.xsiSchemaLocationSplit` setting with experimental formatter. See [#1246](https://github.com/eclipse/lemminx/issues/1246).
+ * Support `xml.format.spaceBeforeEmptyCloseTag` setting with experimental formatter. See [#1245](https://github.com/eclipse/lemminx/issues/1245).
+ * Support `xml.format.joinContentLines` setting with experimental formatter. See [#1244](https://github.com/eclipse/lemminx/issues/1244).
+ * Support `xml.format.joinCommentLines` setting with experimental formatter. See [#1243](https://github.com/eclipse/lemminx/issues/1243).
+ * Support `xml.format.preserveEmptyContent` setting with experimental formatter. See [#1242](https://github.com/eclipse/lemminx/issues/1242).
+ * Support `xml.format.joinCDATALines` setting with experimental formatter. See [#1241](https://github.com/eclipse/lemminx/issues/1241).
+ * Support `xml.format.preservedNewlines` setting with experimental formatter. See [#1240](https://github.com/eclipse/lemminx/issues/1240).
+ * Support `xml.format.enforceQuoteStyle` setting with experimental formatter. See [#1239](https://github.com/eclipse/lemminx/issues/1239).
+ * Support `xml.format.emptyElements` setting with experimental formatter. See [#1238](https://github.com/eclipse/lemminx/issues/1238).
+ * XML validation errors of "Language Servers" are only partially visible. See [#1145](https://github.com/eclipse/lemminx/issues/1145).
+ * Validate uri of XML catalog. See [#823](https://github.com/eclipse/lemminx/issues/823).
+ * CodeAction/Completion to insert required attributes from XSD. See [#365](https://github.com/eclipse/lemminx/issues/365).
+
+### Bug Fixes
+
+ * Checking base complex type for annotations. See [#1268](https://github.com/eclipse/lemminx/pull/1268).
+ * Autocompletion of attribute values won't display documentation. See [#1260](https://github.com/eclipse/lemminx/pull/1260).
+ * Format CDATA sections without adding new lines. See [#1193](https://github.com/eclipse/lemminx/issues/1193).
+
+### Build
+
+ * Unit tests should not store data in ~/.lemminx. See [#1265](https://github.com/eclipse/lemminx/issues/1265).
```

Comment:
`~` is pretty Linux specific so maybe we should use `$HOME` to make it more obvious across all platforms ? Could also just say ".. in the .lemminx folder under the user's home directory."

Reply:
fixed